aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -263,6 +263,12 @@ module aptos_framework::stake {
         borrow_global<StakePool>(pool_address).locked_until_secs
     }
 
+    /// Return the remaining lockup of the stake pool at `pool_address`.
+    /// This will throw an error if there's no stake pool at `pool_address`.
+    public fun get_remaining_lockup_secs(pool_address: address): u64 acquires StakePool {
+        borrow_global<StakePool>(pool_address).locked_until_secs - timestamp::now_seconds()
```

Comment:
Is it going to fail if called after the lockup ends? Isn't it safer to use `get_lockup_secs()` directly? Maybe do it into the if-else with zero as a fallback?